samples/account-linking/csharp/.tours/conversational-account-linking.tour

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Conversational Account Linking",
+  "steps": [
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
```

Comment:
Tried testing this locally, you need to update these paths so they are releative to the root of the repo. Or at least, as is didn't work for me.

Reply:
Updated the paths / line nums. _should_ be up to date now. Realizing code review & updates aren't the easiest with these tours. Definitely has room for improvement.